proposals/bip-002.md

```diff
@@ -47,7 +47,9 @@ marking the incompleteness of this information to the UAST user.
 This change will add two new roles the UAST definition:
 
 - `Incomplete`: For nodes that have been annotated but the annotations don't
-  express the full semantic meaning.
+  express the full semantic meaning or there aren't any other annotations because
```

Comment:
I like it. I tend to be too verbose, yes. Also reworded the `Unannotated` description a little:

For nodes that have received no other annotations. Automatically
 added by the SDK. Driver's developers should aim to have none of these.*